src/components/layout/app-layout/components/header-link.tsx

```diff
@@ -1,20 +1,34 @@
 import { NavLink } from "react-router-dom";
 import {cn} from "@/theme/utils.ts";
 
+import { DropdownMenuIcon } from "./dropdown-menu-icon";
+
 type Props = {
   text: string;
   location: string;
+  subtitle?: string;
+  isDropdown?: boolean;
 }
 
 export function HeaderLink(props: Props) {
+  const { text, location, subtitle, isDropdown } = props;
+
   return (
-    <li>
-      <NavLink to={props.location} className={({isActive}) => cn([
-        'block transition hover:text-primary/80 md:px-4',
-        isActive && `text-primary`
-      ])}>
-        {props.text}
-      </NavLink>
+    <li className={`${isDropdown && 'w-64 p-2 rounded-lg hover:bg-slate-100'}`}>
+      <div className={'flex items-center'}>
+        <div>
+          {isDropdown && <DropdownMenuIcon feature={text} />}
```

Comment:
You shouldn't decide the icon base in text. What happens if we add translations?

You should pass this icon as a prop. Maybe with a default if not specified